ADALiOS/ADALiOS/ADWebRequest.m

```diff
@@ -237,9 +241,12 @@ - (NSURLRequest *)connection:(NSURLConnection *)connection willSendRequest:(NSUR
 {
 #pragma unused(connection)
 #pragma unused(redirectResponse)
+    NSURL* requestURL = [request URL];
+    NSURL* modifiedURL = [ADHelpers addClientVersionToURL:requestURL];
+    if (modifiedURL == requestURL)
```

Comment:
That == is pointer equality, if the pointer didn't change then I didn't change the URL. There's no operator overloading in ObjC